### ISSUE TITLE
Ubuntu 16.04 docker daemon not started

### ISSUE THREAD

**Issue by SvenDowideit** [docker/machine#3950]
I'm using https://github.com/packethost/docker-machine-driver-packet

and when I tell it to use ubuntu-16-04, docker-machine gets to provisioning Docker, but doesn't `systemctl start docker` (or fails doing it, IDK yet).

so docker-machine just loops forever, or until i `docker-machine ssh <host> systemctl start docker`

**Comment by nathanleclaire** [comment]
yeah, i think the daemon doesn't work to start up, i've gotten a few reports like this, looking into it

**Comment by mostafahussein** [comment]
is this a problem with ubuntu 16 specifically ? because yester day i was working on ubuntu 15 with no problems then i decided to start using ubuntu 16 but i haven't test it yet.

**Comment by nathanleclaire** [comment]
It seems to be an issue with Packet specifically, I can create 16.04 nodes fine on DigitalOcean

**Comment by SvenDowideit** [comment]
@crunchywelch ?

**Comment by crunchywelch** [comment]
yeah, I've seen this happen before with our ubuntu hosts, I think it is an issue with our mirrors, but needs some more investigation. @SvenDowideit do you have any cycles to dig a bit deeper? if not, I'll try to check it this week
